content/en/platform/corda/4.5/enterprise/operations/deployment/deployment-kubernetes-farm.md

```diff
@@ -34,8 +34,8 @@ helm install cenm-farm farm --set prefix=cenm --set acceptLicense=Y --set volume
 | Parameter                     | Description                                              | Default value         |
 | ----------------------------- | -------------------------------------------------------- | --------------------- |
 | `bashDebug`                   | Display additional information while running bash scripts (useful while investigating issues) | `false` |
-| `farmImage.repository`        | URL to FARM Docker image repository                      | `acrcenm.azurecr.io/auth/auth` |
-| `farmImage.tag`               | Docker image tag | `1.3` |
+| `farmImage.repository`        | URL to FARM Docker image repository                      | `corda/enterprise-auth` |
```

Comment:
This should point to the Farm image instead. Can we also fix the uppercase service name in the description?